preview-image.js

```diff
@@ -0,0 +1,13 @@
+const puppeteer = require('puppeteer')
+
+;(async () => {
```

Comment:
I just found out there is such a way 😁

Reply:
Haha spying on our work in progress. Yes there is always a way 😄